docs/_layouts/docs.html

```diff
@@ -14,12 +14,12 @@
     {% include top_nav.html %}
 
     <div class="container">
-      <div class="row">
-        <div class="sidebar col-md-3 col-sm-12">
+      <div class="grid-l-col">
```

Comment:
Question! why did we move to grid instead of bootstrap?

Reply:
I wanted to keep the sidebar at a consistent width. Could use Bootstrap for this with some overrides, but anything to reduce dependencies on external resources is good imo. That said, we need to add in some fallbacks for older browsers that don't support css-grid.